src/pages/veiligheidsregio/index.tsx

```diff
@@ -76,10 +85,13 @@ const SafetyRegion: FCWithLayout<any> = () => {
           <h2 className="text-max-width">
             {text.veiligheidsregio_index.selecteer_titel}
           </h2>
-          <p className="text-max-width">
-            {text.veiligheidsregio_index.selecteer_toelichting}
-          </p>
-          <EscalationMapLegenda />
+          <div
+            className="text-max-width"
+            dangerouslySetInnerHTML={{
+              __html: text.veiligheidsregio_index.selecteer_toelichting,
+            }}
+          />
+          {isLargeScreen && <EscalationMapLegenda text={text} />}
```

Comment:
```suggestion
          <EscalationMapLegenda text={text} />
```
Don't render legend conditionally. I think this is a merge error probably.

Reply:
it is